benches/reader.rs

```diff
@@ -0,0 +1,111 @@
+#![feature(test)]
+extern crate grcov;
+// extern crate rustc_hash;
+extern crate test;
+
+use grcov::{GCNO, GcovReaderBuf};
+use std::path::PathBuf;
+use test::{black_box, Bencher};
+
+
+#[bench]
+fn bench_reader_gcno(b: &mut Bencher) {
+    let mut gcno = GCNO::new();
+    b.iter(|| black_box(gcno.read(GcovReaderBuf::from("test/llvm/reader.gcno"))
+        .unwrap()));
+}
+
+#[bench]
+fn bench_reader_gcno_gcda(b: &mut Bencher) {
+    let mut gcno = GCNO::new();
+    gcno.read(GcovReaderBuf::from("test/llvm/reader.gcno"))
+        .unwrap();
+    
+    b.iter(|| black_box(gcno.read_gcda(GcovReaderBuf::from("test/llvm/reader.gcda"))
+        .unwrap()));
+}
+
+#[bench]
+fn bench_reader_gcno_gcda_gcda(b: &mut Bencher) {
+    let mut gcno = GCNO::new();
+    gcno.read(GcovReaderBuf::from("test/llvm/reader.gcno"))
+        .unwrap();
+
+    b.iter(|| {
+        for _ in 0..2 {
```

Comment:
Why twice?

Reply:
I based it off of this [test](https://github.com/mozilla/grcov/blob/fd147e33b644b8d4422d07254f74462024541d20/src/reader.rs#L1001). I am not sure how should I be doing this one?

Reply:
OK, that's mostly a correctness test, so no need for this benchmark.